packages/data/addon/adapters/bard-facts-v2.ts

```diff
@@ -12,52 +12,10 @@ import { assign } from '@ember/polyfills';
 import EmberObject from '@ember/object';
 import { canonicalizeMetric, serializeParameters, getAliasedMetrics, canonicalizeAlias } from '../utils/metric';
 import { configHost } from '../utils/adapter';
+import NaviFactAdapter, { Filter, Parameters, RequestOptions, RequestV2, SORT_DIRECTIONS } from './fact-interface';
 
-const SORT_DIRECTIONS = ['desc', 'asc'];
-
-export type RequestOptions = {
-  clientId?: string;
-  customHeaders?: Dict<string>;
-  timeout?: number;
-  page?: number;
-  perPage?: number;
-  format?: string;
-  cache?: boolean;
-  queryParams?: Dict<string>;
-  dataSourceName?: string;
-};
-type Query = RequestOptions & Dict<string | number | boolean>;
-type SortDirection = typeof SORT_DIRECTIONS[number];
-type ColumnType = 'metric' | 'dimension' | 'timeDimension';
-type Parameters = Dict<string>;
-type Column = {
-  field: string;
-  parameters: Parameters;
-  type: ColumnType;
-  alias: string;
-};
-type Filter = {
-  field: string;
-  parameters: Parameters;
-  type: ColumnType;
-  operator: string;
-  values: string[];
-};
-type Sort = {
-  field: string;
-  parameters: Parameters;
-  direction: SortDirection;
-};
-// TODO: Remove V2 once V1 is no longer in use
-type RequestV2 = {
-  filters: Filter[];
-  columns: Column[];
-  table: string;
-  sort: Sort[];
-  limit: TODO<string>;
-  requestVersion: '2.0';
-};
-type AliasFn = (column: string) => string;
+export type Query = RequestOptions & Dict<string | number | boolean>;
+export type AliasFn = (column: string) => string;
```

Comment:
we actually don't need that here since we're not going to add aliases like in v1 `sort` and `having` but we can save this cleanup for separate PR

Reply:
We actually have this shape in v1:
https://github.com/yahoo/navi/pull/849/files#diff-77567338607c3ace754f83a149f0c9e7R127

Reply:
I mean, should be _only_ in v1